src/views/Playlist/Playlist.tsx

```diff
@@ -22,16 +22,23 @@ const getPlaylistCodeFromUrl = (pathname: string) => {
   return roomCode
 }
 
-const getPlaylistSongs = async (roomCode: string, setPlaylistSongs: Function) => {
+const getPlaylistSongs = async (roomCode: string, offset: number) => {
   const url = new URL('/playlist/tracks', process.env.REACT_APP_SERVER_URL)
   url.searchParams.append('room', roomCode)
+  url.searchParams.append('offset', String(offset))
 
   try {
     const result = await fetch(url.href, {
       method: 'get'
     })
     const json = await result.json()
-    setPlaylistSongs(json.items)
+    if (json.next) {
+      const next = await getPlaylistSongs(roomCode, offset+100)
+      json.items = [...json.items, ...next.items]

```

Comment:
Instead of tallying them up and then adding them to the playlist at the end, we can append songs to the playlist as we get them. This would give the end user information quicker.

Reply:
Im working on something for that now

Reply:
https://github.com/CSSmitty/Spoofity_Client/pull/17